src/__stories__/tag-story.tsx

```diff
@@ -1,42 +1,139 @@
 import * as React from 'react';
-import {Stack, Tag, useTheme, Text1} from '..';
-import {useTextField} from './helpers';
+import {
+    Stack,
+    Tag,
+    useTheme,
+    Text3,
+    IconStarFilled,
+    IconTimeRegular,
+    IconAlarmLightRegular,
+    IconLikeRegular,
+    IconSkullRegular,
+    IconSmileySadRegular,
+    Inline,
+    Text1,
+} from '..';
+import {ThemeVariant} from '../theme-variant-context';
+
+type Args = {
```

Comment:
Is it possible to show list of args in playroom?
![image](https://user-images.githubusercontent.com/6722153/143083076-1f542334-1055-48f4-96b5-257b7def6530.png)

Reply:
oki